pkgs/misc/cups-pdf/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, fetchurl, cups, ghostscript }:
```

Comment:
Does it make more sense to put this under misc/cups/drivers?

Reply:
Yes, that makes more sense.